server/service/http/message/query/query_structure.rs

```diff
@@ -83,12 +83,20 @@ pub enum QueryStructureEdgeTypeResponse {
     Argument(String),
 }
 
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct QueryStructureEdgeSpan {
```

Comment:
Looks ok, though only thing i wonder is if the end is inclusive or exclusive

Reply:
inclusive, as in TypeQL. Which is quite confusing.

Reply:
ok